ts/src/binanceus.ts

```diff
@@ -100,6 +102,106 @@ export default class binanceus extends binance {
                 'private': {
                     'get': {
                         'status': 1,
+                        'rateLimit/order': 20,
+                        'order': 2,
+                        'openOrders': { 'cost': 3, 'noSymbol': 40 },
+                        'myTrades': 10,
+                        'myPreventedMatches': 10, // with ID it has weight 1, but we don't have that complex handling yet
+                        'allOrders': 10,
+                        'orderList': 2,
+                        'allOrderList': 10,
+                        'openOrderList': 3,
+                    },
+                    'post': {
+                        'order': 1,
+                        'order/test': 1,
+                        'order/cancelReplace': 1,
+                        'order/oco': 1,
+                    },
+                    'delete': {
+                        'order': 1,
+                        'openOrders': 1,
+                        'orderList': 1,
+                    },
+                },
+                'sapi': {
```

Comment:
I'm getting some "not found" errors

```
n binanceus sapiGetApiTradingStatus
Debugger attached.
2024-01-26T09:45:18.178Z
Node.js: v20.7.0
CCXT v4.2.21
binanceus.sapiGetApiTradingStatus ()
ExchangeNotAvailable binanceus GET https://api.binance.us/sapi/v1/apiTradingStatus?timestamp=1706262319131&recvWindow=10000&signature=803944d95d6cbad84cccb5102deabaf549bfb195f28f6f67ea7a1505e51a1d6a 404  {"timestamp":1706262319414,"status":404,"error":"Not Found","message":"No message available","path":"/sapi/v1/apiTradingStatus"}
```
```
n binanceus privateGetStatus                       
Debugger attached.
2024-01-26T09:45:51.849Z
Node.js: v20.7.0
CCXT v4.2.21
binanceus.privateGetStatus ()
ExchangeNotAvailable binanceus GET https://api.binance.us/api/v3/status?timestamp=1706262352827&recvWindow=10000&signature=2326ef3588894c352107b7d2d6e93303a379ec85d37e6598bad7c1ac455c9b97 404 Not Found <html><body><h2>404 Not found</h2></body></html>
```

Reply:
eventually, i had to revise all endpoints, as I had several misplaced ones. now all corrected.

Reply:
going to check it